templates/service/conf/messages.cy

```diff
@@ -0,0 +1,12 @@
+service.name = $!APP_NAME!$
+service.homePageUrl = /$!APP_NAME!$
+service.text = Dyma'ch gwasanaeth newydd
+
+footer.cookies.text = Cwcis
```

Comment:
We shouldn't need these footer messages as they are part of play-frontend-hmrc.

Reply:
@matthewmascord Removed, along with the English versions.